server/src/main/java/org/elasticsearch/rest/RestHandler.java

```diff
@@ -173,7 +177,7 @@ private Route(
          * @param path   the path, e.g. "/"
          */
         public Route(Method method, String path) {
-            this(method, path, RestApiVersion.current(), null, null, null);
+            this(method, path, RestApiVersion.current(), null, Level.OFF, null);
```

Comment:
we effectively ignore Level.OFF, but it more expressive than passing around `null` (which used to have meaning based on the other context .. here even though it is not used, it should be expressive)